packages/dev-utils/package.json

```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "@backstage/core": "^0.5.0",
     "@backstage/catalog-model": "^0.7.0",
-    "@backstage/plugin-catalog": "^0.2.12",
+    "@backstage/plugin-catalog-react": "^0.0.1",
```

Comment:
This dependency of a core package to a plugin is strange. Maybe we can move the helper to `plugin-catalog-react`?